Makefile

```diff
@@ -7,6 +7,7 @@
     build_mysql_integration_test_images clean_integration_test_images \
 	dm dm-master dm-worker dmctl dm-syncer dm_coverage \
 	engine tiflow tiflow-demo tiflow-chaos-case help
+.DEFAULT_GOAL := default
```

Comment:
1. Please refine the indent of L7 and keep the same with other lines.
L7 uses four spaces and other lines use tab.

2. It's better to insert a blank line before the `.DEFAULT_GOAL := default`.

Rest LGTM

Reply:
1. It's an issue with GitHub.
![image](https://user-images.githubusercontent.com/29879298/182570237-74006d6c-e261-4321-a2c1-472f803d4953.png)
2. I think it makes sense because of almost all PHONY in here.

Reply:
<img width="708" alt="image" src="https://user-images.githubusercontent.com/1638682/182573288-580c2d9b-62cb-4ef7-b643-37a43cd1f7c4.png">